src/sonic-daemon-base/sonic_daemon_base/daemon_base.py

```diff
@@ -18,6 +18,9 @@
 # Platform root directory inside docker
 PLATFORM_ROOT_DOCKER = '/usr/share/sonic/platform'
 SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+SONIC_ENV_FILE = '/etc/sonic/sonic-environment'
+PLATFORM_ENVVAR = 'PLATFORM'
+HWSKU_ENVVAR = 'HWSKU'
```

Comment:
This file shouldn't even be calling sonic-cfggen in the first place. This problem will be remediated by my PR here: https://github.com/Azure/sonic-buildimage/pull/5003

Reply:
That awesome! one file less!